ComponentKit/HostingView/CKComponentHostingView.h

```diff
@@ -16,6 +16,11 @@
 @protocol CKComponentHostingViewDelegate;
 @protocol CKComponentSizeRangeProviding;
 
+typedef NS_ENUM(NSUInteger, CKComponentHostingViewMode) {
```

Comment:
`CKComponentHostingViewUpdateMode` might be clearer. Could this be unified with the enum in the transactional data source?